### PR TITLE
[haml/de] Add Haml/de article

### PR DESCRIPTION
**Pull request by s-ol**

### CODE REVIEW

File: de-de/haml-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,156 @@
+---
+language: haml
+filename: learnhaml.haml
```
Comment (vendethiel) [inline de-de/haml-de.html.markdown]: please add a `-de` suffix

Comment (s-ol) [inline de-de/haml-de.html.markdown]: @vendethiel done

**Comment by vendethiel** [comment]
Thanks!

**Comment by vendethiel** [comment]
:+1:

**Pull request merged**